go.mod

```diff
@@ -1,31 +1,196 @@
 module github.com/textileio/go-libp2p-pubsub-rpc
 
-go 1.16
+go 1.17
 
 require (
 	github.com/gopherjs/gopherjs v0.0.0-20190812055157-5d271430af9f // indirect
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d // indirect
-	github.com/hsanjuan/ipfs-lite v1.1.23
+	github.com/hsanjuan/ipfs-lite v1.2.1-0.20220309173135-e3246b1d1b2f
 	github.com/ipfs/go-cid v0.1.0
-	github.com/ipfs/go-ipfs-blockstore v1.0.4
+	github.com/ipfs/go-ipfs-blockstore v1.1.2
 	github.com/ipfs/go-ipfs-config v0.16.0
 	github.com/ipfs/go-ipfs-util v0.0.2
-	github.com/ipfs/go-ipld-cbor v0.0.5
+	github.com/ipfs/go-ipld-cbor v0.0.6
 	github.com/ipfs/go-ipld-format v0.2.0
-	github.com/libp2p/go-libp2p v0.15.1
-	github.com/libp2p/go-libp2p-connmgr v0.2.4
-	github.com/libp2p/go-libp2p-core v0.9.0
-	github.com/libp2p/go-libp2p-peerstore v0.3.0
+	github.com/libp2p/go-libp2p v0.17.0
+	github.com/libp2p/go-libp2p-connmgr v0.3.1
+	github.com/libp2p/go-libp2p-core v0.13.0
+	github.com/libp2p/go-libp2p-peerstore v0.6.0
 	github.com/libp2p/go-libp2p-pubsub v0.5.5
-	github.com/libp2p/go-libp2p-quic-transport v0.11.2
-	github.com/multiformats/go-multiaddr v0.4.1
+	github.com/libp2p/go-libp2p-quic-transport v0.15.2
+	github.com/multiformats/go-multiaddr v0.5.0
 	github.com/smartystreets/assertions v1.0.1 // indirect
 	github.com/stretchr/testify v1.7.0
-	github.com/textileio/go-ds-badger3 v0.0.0-20210324034212-7b7fb3be3d1c
+	github.com/textileio/go-ds-badger3 v0.1.0
```

Comment:
This is using a new tag created after https://github.com/textileio/go-ds-badger3/pull/1, which solves the breaking API changes in `go-datastore`.